### PR TITLE
Add option to toggle to new engine for legacy simulator

### PR DESCRIPTION
**Pull request by ericnewcomer**
### What it does
* Switch back and forth using `/v1` and `/v2` in the simulator
* Only displays events taken by the new engine, no db objects are created
* Adds webhook results in a dialog instead of a db object read page

### What it doesn't do (yet)
* Does not support simulating flows not currently supported by the new engine (surveyor, ussd)
* Does not currently honor keyword triggers into other flows from the current simulation

### CODE REVIEW

File: temba/settings_common.py [hunk 1]
```diff
@@ -1223,6 +1224,14 @@
 # -----------------------------------------------------------------------------------
 COURIER_CHANNELS = set(['CT', 'DK', 'MT', 'WA', 'ZV'])
 
+# -----------------------------------------------------------------------------------
```
Comment (rowanseymour) [inline temba/settings_common.py]: is this is a weird merge or why is this appearing as added lines?

Comment (rowanseymour) [reply temba/settings_common.py]: actually seems all the new_engine stuff is appearing as new so just needs updated against master..

Comment (ericnewcomer) [reply temba/settings_common.py]: It's actually up to date. It looks like goflow was converted into a module and it goobered my merge. I'll repair things and let you know when to have another look.

Comment (ericnewcomer) [reply temba/settings_common.py]: better.

File: temba/flows/tests.py [hunk 2]
```diff
@@ -6929,12 +6956,12 @@ def test_translations(self):
         # test that simulation takes language into account
         self.login(self.admin)
         simulate_url = reverse('flows.flow_simulate', args=[favorites.pk])
-        response = json.loads(self.client.post(simulate_url, json.dumps(dict(has_refresh=True)), content_type="application/json").content)
+        response = json.loads(self.client.post(simulate_url, json.dumps(dict(has_refresh=True, version="1")), content_type="application/json").content)
```
Comment (rowanseymour) [inline temba/flows/tests.py]: @dodobas weren't these switched to .json() in your past PR? Just checking we're not re-introducing python 3 incompatibilities whilst we still have no Python 3 CI tests

File: temba/flows/views.py [hunk 3]
```diff
@@ -1128,6 +1130,40 @@ def post(self, request, *args, **kwargs):
...
+                # simulating never caches
+                asset_timestamp = int(time.time() * 1000000)
+                flow = self.get_object(self.get_queryset())
+
+                # we control the pointers to ourselves and environment ignoring what the client might send
+                flow_request = client.request_builder(asset_timestamp).asset_server(flow.org).set_environment(flow.org)
+
+                # when testing, we need to include all of our assets
+                if settings.TESTING:
+                    flow_request.include_all(flow.org)
```
Comment (rowanseymour) [inline temba/flows/views.py]: not a problem for this PR.. but would be nice to have a cleaner solution here. Maybe a suite of non-transactional tests for testing the flowserver

**Comment by ericnewcomer** [comment]
Ok @rowanseymour, I merged in changes for new goflow events and updated the new_sim stuff to work. Latest goflow running locally is taking way longer to handle these requests and one of my test flows that worked before no longer does, so something suspicious there.

**Pull request merged**